ott/core/sinkhorn_lr.py

```diff
@@ -567,6 +663,18 @@ def make(
       use_danskin=use_danskin,
       implicit_diff=implicit_diff,
       jit=jit,
-      rng_key=rng_key,
       kwargs_dys=kwargs_dys
   )
+
+
+# TODO(michalk8): move to math utils
+def kl(q1: jnp.ndarray, q2: jnp.ndarray) -> float:
+  res_1 = -jsp.special.entr(q1)
+  res_2 = q1 * safe_log(q2)
```

Comment:
have you considered 

https://jax.readthedocs.io/en/latest/_autosummary/jax.scipy.special.xlogy.html

it's also safe (as long as x ~ y ~ 0)

Reply:
It's only safe in `x ~ 0`, not `y ~ 0`:

```python
import jax.numpy as jnp
import ott

x = jnp.ones(3)
y = jnp.ones(3).at[0].set(0) / 2

jax.scipy.special.xlogy(x, y)
# DeviceArray([      -inf, -0.6931472, -0.6931472], dtype=float32)

x * ott.core.sinkhorn_lr.safe_log(y)
# DeviceArray([-87.33655  ,  -0.6931472,  -0.6931472], dtype=float32)
```